src/main/resources/sentry-sdk.properties

```diff
@@ -0,0 +1 @@
+sdk_version = 6.25.2
```

Comment:
```suggestion
sdk_version = 6.31.0
```
`m` or do we want to keep it low so we can see the updater in action after merging?

Reply:
yep, would like to keep it low to test the updater when we merge